css-brochure-form/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h2>LEARN MORE</h2>
+  <section>
+    <form>
+      <div class="email">
+        <label for="email">COMPANY EMAIL</label>
+        <div>
+          <input type="email" id="email" placeholder="sally@company.co">
+        </div>
+      </div>
+      <div class="name">
+        <label for="name">NAME</label>
+        <div>
+          <input type="text" id="name" placeholder="Sally Smith">
+        </div>
+      </div>
+      <div class="where">
+        <label for="where">WHERE ARE YOU HIRING</label>
+        <div>
+          <input type="text" id="where" placeholder="City, Country">
+        </div>
+      </div>
+      <div class="engineers">
+        <label for="engineers">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS</label>
+        <div>
+          <select id="engineers">
+            <option value="1-2">1-2 Engineers</option>
+            <option value="3-4">3-4 Engineers</option>
+            <option value="5-6">5-6 Engineers</option>
+          </select>
+          <div class="button">
+            <button type="submit" name="submit" value="Submit">SUBMIT
```

Comment:
Couple of things here:
1. When `button` is within `form` we don't have to specify which `type` it is. it's `type="submit"` by its default when it's inside of `form`
2. We do not need `name` and `value` for the button. Those two are usually for server to see which data it is for. But since button doesn't have any data, server doesn't need to know the name or value.
3. `button` is not self-closing tag so we are missing a closing here :)